lib/src/follower.dart

```diff
@@ -258,6 +260,13 @@ class RenderFollowerLayer extends RenderProxyBox {
 
     if (!link.leaderConnected && _previousFollowerOffset == null) {
       print("The leader isn't connected and there's no cached offset. Not painting anything.");
+      // In the first frame we are not connected to the leader.
+      // Check again in the next frame.
+      WidgetsBinding.instance.addPostFrameCallback((timeStamp) {
```

Comment:
Is there another signal that we could use for this purpose? It's possible that we're never linked, which would result in continuous paint requests for no reason....

Reply:
Updated.

Reply:
This still seems fragile and overly specific. I see that I copied the implementation of `LayerLInk`, which means we can change it. Would this problem be solved by having `LayerLink` notify listeners when a leader is connected or disconnected?

Reply:
It seems that we are constantly changing from connected to disconnected. With this change, the test timed out. Relevant code:

```dart
set leader(CustomLeaderLayer? value) {
    if (value == _leader) {
      return;
    }
    final previousLeaderConnected = leaderConnected;
    _leader = value;
    if (leaderConnected != previousLeaderConnected) {
      notifyListeners();
    }
  }
```

Reply:
Can you try to found out why we're switching from connected to disconnected? This sounds like a bug that we should fix.

Reply:
This is only happening with the new code with a listener. In the `Leader` paint method, when we call `pushLayer` it causes `detach` and `attach` to be called on the `CustomLeaderLayer`. These methods update the `leader` in the `LayerLink`, which notifies the listener again...

Reply:
Unfortunately, It doesn't seem that we have a way to avoid that.